templates/collection.nomad

```diff
@@ -92,9 +92,39 @@ job "collection-${name}" {
           pg = 5432
         }
       }
-      env {
-        POSTGRES_USER = "snoop"
-        POSTGRES_DATABASE = "snoop"
+      template {
+        data = <<EOF
+          POSTGRES_USER = "snoop"
+          POSTGRES_DATABASE = "snoop"
+          {{- with secret "liquid/collections/${name}/snoop.postgres" }}
+            POSTGRES_PASSWORD = {{.Data.secret_key}}
```

Comment:
Shouldn't this be in quotes like the ones above it? I guess the `env = true` files get parsed by some golang library.

If we have been using this with and without them around the code base everywhere, we probably really need to fix that. Could you put quotes around everything in your patch?

Thanks!

Reply:
Yeah you're right about that and I have to admit that I'm not always consistent with it. I'll change that.

Reply:
The nomad people use `| toJSON`for variables which may contain special characters. Also, they don't put quotes around their example:
`DB_PASSWD={{ .Data.data.DB_PASSWD | toJSON }}`
Otherwise they seem to put them in quotes like this:  
`LOG_LEVEL="{{key "service/geo-api/log-verbosity"}}"`
So I can pipe the password to `toJSON`, but it wouldn't make sense to do the same with the hardcoded names and stuff. I'll make it consistent within the altered section so that environemnt variables are in quotes and there are no spaces between the name and the assignment, just like in the nomad example.

Reply:
`toJSON` adds `" "` because that's the JSON way of encoding a string. It makes sense to always use it when getting strings from Consul and from Vault.

Your example is just a sign that we're not the only ones with a consistency problem for things like these.